packages/babel-parser/test/fixtures/estree/typescript/instantiation-expression-optional-chain-babel-7/output.json

```diff
@@ -0,0 +1,112 @@
+{
+  "type": "File",
+  "start":0,"end":19,"loc":{"start":{"line":1,"column":0},"end":{"line":2,"column":10}},
+  "program": {
+    "type": "Program",
+    "start":0,"end":19,"loc":{"start":{"line":1,"column":0},"end":{"line":2,"column":10}},
+    "sourceType": "script",
+    "interpreter": null,
+    "body": [
+      {
+        "type": "ExpressionStatement",
+        "start":0,"end":8,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":8}},
+        "expression": {
+          "type": "TSInstantiationExpression",
+          "start":0,"end":7,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":7}},
+          "expression": {
+            "type": "ChainExpression",
+            "start":0,"end":7,"loc":{"start":{"line":1,"column":0},"end":{"line":1,"column":7}},
```

Comment:
The `ChainExpression` should end at the start of the `TSTypeParameterInstantiation`.